### PR TITLE
chore(docs): make version support more explicit

### PR DESCRIPTION
**Pull request by stirby**

### CODE REVIEW

File: docs/install/releases.md [hunk 1]
```diff
@@ -7,12 +7,13 @@ We recommend enterprise customers test the compatibility of new releases with
...
 deployment.
 
-We support two release channels:
-[mainline](https://github.com/coder/coder/releases/tag/v2.13.0) for the bleeding
-edge version of Coder and
-[stable](https://github.com/coder/coder/releases/latest) for those with lower
-tolerance for fault. We field our mainline releases publicly for one month
-before promoting them to stable.
+We support three releases of Coder concurrently. We release
+[mainline](https://github.com/coder/coder/releases/tag/v2.13.0) as the bleeding
```
Comment (EdwardAngert) [inline docs/install/releases.md:11]:
```suggestion
[mainline](https://github.com/coder/coder/releases/tag/v2.15.0) as the bleeding
```

per <https://github.com/coder/coder/releases/>

Comment (stirby) [reply docs/install/releases.md:11]: good catch, we should make this update automatically

Comment (stirby) [reply docs/install/releases.md]: fixed statically for now.

**Review by EdwardAngert — APPROVED** [review]
🚀

File: docs/install/releases.md [hunk 2]
```diff
@@ -7,12 +7,13 @@ We recommend enterprise customers test the compatibility of new releases with
...
 deployment.
 
-We support two release channels:
-[mainline](https://github.com/coder/coder/releases/tag/v2.13.0) for the bleeding
-edge version of Coder and
-[stable](https://github.com/coder/coder/releases/latest) for those with lower
-tolerance for fault. We field our mainline releases publicly for one month
-before promoting them to stable.
+We support three releases of Coder concurrently. We release
+[mainline](https://github.com/coder/coder/releases/tag/v2.15.0) as the bleeding
```
Comment (matifali) [inline docs/install/releases.md:11]:
```suggestion
[mainline](https://github.com/coder/coder/releases/tag/v2.16.0) as the bleeding
```

**Pull request merged**